Update experiment-management-api-experiments.md (#497)
amplitude/amplitude-docs: content/collections/experiment-apis/en/experiment-management-api-experiments.md

## Edit

{{partial:admonition type='note'}}

> Both web experiments and guides & surverys experiments do not support the editing of experiments. Attempts to edit a web experiment or a guides & surverys experiment return a `405` error.

Reviewer: ```suggestion
Neither Web Experimentation or Guides and Surveys support editing experiments. Attempts to edit a web experiment or a guides and surveys experiment return a `405` error.
```
